### ISSUE TITLE
Messages stuck on encrypting message

### ISSUE THREAD

**Issue by EviL-GaMer** [loki-project/session-android#158]
Have been playing around with Session to try it as a secure chat tool. It worked for a while but now every message I send to no matter who gets stuck on "encrypting message" and then doesn't get sent.

I can still receive messages but not send. Clearing the cache doesn't do anything, only clearing the storage and creating a new ID helped.

Basically all I did was chat and then the behaviour appeared. With some chats it took more time than with others but in the end they all stopped sending.

 - Device: Xiaomi Redmi Note 4
 - OS: Resurrection Remix v7.0.2-20190916 (Android 9)
 - Version of Loki Messenger or latest commit hash: 1.0.10

I am also running ExpressVPN if that matters. Turning it off didn't change anything and I don't think it's connected as it's stuck at encrypting and not path tracing.

**Comment by Brice-W** [comment]
I ran into this issue while testing something else. It definitely appears after I installed v1.0.10 on my Android devices (phone and Virtual device). I have a similar behaviour: message not sending, blocking at "encrypting message" stage, but I can still receive messages.

**Comment by Mikunj** [comment]
> I ran into this issue while testing something else. It definitely appears after I installed v1.0.10 on my Android devices (phone and Virtual device). I have a similar behaviour: message not sending, blocking at "encrypting message" stage, but I can still receive messages.

I've run into this issue on a chat (Note to self) between my linked devices. Does this also apply to you or are you getting it with chatting with other users?

**Comment by Brice-W** [comment]
> I've run into this issue on a chat (Note to self) between my linked devices. Does this also apply to you or are you getting it with chatting with other users?

Yes I tried with the Note to self conversation, I am also having the issue in that case (I did 2 tests: Android linked to iOS and Android linked to Desktop, both stopped on "encrypting message")
<img width="288" alt="Screen Shot 2020-04-01 at 4 07 19 pm" src="https://user-images.githubusercontent.com/62363622/78101467-549dc000-7433-11ea-818d-924e94841a42.png">

**Comment by Mikunj** [comment]
If you put a breakpoint on `sendTextMessage` in `ConversationActivity` and evaluate 
```
DatabaseFactory.getLokiAPIDatabase(context).getDeviceLinks(TextsecurePreferences.getLocalNumber(context))
```
Does it show any device links?

**Comment by KeeJef** [comment]
Ran into this yesterday too, just in a normal person to person conversation, hadnt seen this before  10.0.10

**Comment by Mikunj** [comment]
Oh ok thanks, i'll take a look at the diff tomorrow and see if it's something that we broke

**Comment by Brice-W** [comment]
> If you put a breakpoint on `sendTextMessage` in `ConversationActivity` and evaluate
> 
> ```
> DatabaseFactory.getLokiAPIDatabase(context).getDeviceLinks(TextsecurePreferences.getLocalNumber(context))
> ```
> 
> Does it show any device links?

Yes there is one

**Comment by datgb** [comment]
Same issue here. I can't communicate with nearly anyone. 1 message goes thru and everything else stuck on encrypting.

**Comment by Mikunj** [comment]
I've located the core of the issue, a fix will be out in the next update :)

**Comment by datgb** [comment]
@Mikunj is there an idea of when that will roll out? Its unreliable for me right now to use. I'll need to ask everyone back to another messenger in meantime. Right in middle of chats for work I can't even reply to anyone asking me questions.

**Comment by Mikunj** [comment]
> @Mikunj is there an idea of when that will roll out? Its unreliable for me right now to use. I'll need to ask everyone back to another messenger in meantime. Right in middle of chats for work I can't even reply to anyone asking me questions.

We've released a patch on version 1.0.11. Give that a go and let us know if that fixes the issue :)

**Comment by Brice-W** [comment]
Definitely working better now, thanks!!!!
